xlive/H2MOD/Modules/CustomVariantSettings/CustomVariantSettings.cpp

```diff
@@ -137,12 +136,6 @@ namespace CustomVariantSettings
 				WriteValue(Memory::GetAddress(0x17a44b), (BYTE)0x1E);
 			else
 				WriteValue(Memory::GetAddress(0x17a44b), (BYTE)0);
-
-			if (newVariantSettings->forcedFOV != 0)
-			{
-				player_control_set_field_of_view(newVariantSettings->forcedFOV);
-				observer_set_suggested_field_of_view(newVariantSettings->forcedFOV);
```

Comment:
This is still needed since you don't have a function like player_control_get_field_of_view for observer cameras.

Reply:
fixed